functions/classes/class.User.php

```diff
@@ -1374,8 +1374,8 @@ public function auth_passkey ($credentialId = "", $encodedCredential = "", $keyI
         # failure
         if(!isset($this->user->username)) {
             throw new Exception ("Cannot fetch credentials from userid");
-        }
             header('HTTP/1.1 500 Cannot fetch credentials from userid');
```

Comment:
Should this header() be before the throw()?
It's unreachable.

Reply:
Yes you're right